ansible/tasks/setup-supabase-internal.yml

```diff
@@ -80,9 +80,6 @@
 - name: Install Postgres exporter
   import_tasks: internal/postgres-exporter.yml
 
-- name: Install supautils
```

Comment:
If i understand correctly, then supautils also gets installed through the .deb package, right?

Reply:
yup, supautils releases deb package now, similar to pg_graphql.